docker/README.md

```diff
@@ -0,0 +1,254 @@
+# Nuxeo Docker Images
+
+Nuxeo provides a set of Docker images that can be separated into two categories:
+- The [builder](#builder-images) images.
+- The [final](#final-images) images.
```

Comment:
the base image is missing? 
Nuxeo provides ... into 3 categories:
- The builder image that creates a Nuxeo distribution with packages
- The base image that includes all the needed dependencies to run a Nuxeo distribution
- The final image that composes a base and a builder image

Reply:
For me the base one is part of the"builder" category as it is used to build the final one in the multi-stage build (final is FROM base).
But what you suggest also makes sense.